core/analysis/delimited_token_stream.cpp

```diff
@@ -134,50 +141,95 @@ bool parse_json_config(const irs::string_ref& args, std::string& delimiter) {
 /// @brief args is a jSON encoded object with the following attributes:
 ///        "delimiter"(string): the delimiter to use for tokenization <required>
 ////////////////////////////////////////////////////////////////////////////////
-irs::analysis::analyzer::ptr make_json(const irs::string_ref& args) {
+irs::analysis::analyzer::ptr make_vpack(const irs::string_ref& args) {
   std::string delimiter;
-  if (parse_json_config(args, delimiter)) {
+  if (parse_vpack_options(args, delimiter)) {
     return irs::analysis::delimited_token_stream::make(delimiter);
   } else {
     return nullptr;
   }
 }
-
 ///////////////////////////////////////////////////////////////////////////////
 /// @brief builds analyzer config from internal options in json format
 /// @param delimiter reference to analyzer options storage
 /// @param definition string for storing json document with config 
 ///////////////////////////////////////////////////////////////////////////////
-bool make_json_config(const std::string& delimiter, std::string& definition) {
-  rapidjson::Document json;
-  json.SetObject();
-
-  rapidjson::Document::AllocatorType& allocator = json.GetAllocator();
-  // delimiter
-  json.AddMember(
-    rapidjson::StringRef(DELIMITER_PARAM_NAME.c_str(), DELIMITER_PARAM_NAME.size()),
-    rapidjson::Value(rapidjson::StringRef(delimiter.c_str(), delimiter.length())),
-    allocator
-  );
+bool make_vpack_config(const std::string& delimiter, std::string& definition) {
+  VPackBuilder builder;
+  {
+    VPackObjectBuilder object(&builder);
+    {
+      // delimiter
+      std::string tmp(reinterpret_cast<const char*>(delimiter.c_str()), delimiter.size());
```

Comment:
Why we need a temporary here?
`Value` could be constructed from `const string&`  directly